src/views/Authorizations/ViewAuthorization/ViewAuthorization.js

```diff
@@ -0,0 +1,182 @@
+                                </Col>
+                            </FormGroup>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="status-input">Tipo de procedimiento</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="status-input" name="procedure" disabled={true} value={this.state.authorization.type} onChange={this.handleChange} />
+                                </Col>
+                            </FormGroup>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="email-input">Fecha de creación</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="date-created-input" name="date-created" disabled={true} value={new Date(Date.parse(this.state.authorization.created_at)).toLocaleDateString()} onChange={this.handleChange} />
+                                </Col>
+                            </FormGroup>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="select">Creada por</Label>
+                                </Col>
+                                    <Col xs="12" md="9">
+                                        <Input type="text" id="created-by-input" name="created-by" disabled={true} value={`${this.state.authorization.created_by.dni} - ${this.state.authorization.created_by.first_name} ${this.state.authorization.created_by.last_name}`} onChange={this.handleChange} />
+                                </Col>
+                            </FormGroup>
+                            {this.state.authorization.created_for && <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="select">Creada para</Label>
+                                </Col>
+                                    <Col xs="12" md="9">
+                                        <Input type="text" id="created-for-input" name="created-for" disabled={true} value={`${this.state.authorization.created_for.dni} - ${this.state.authorization.created_for.first_name} ${this.state.authorization.created_for.last_name}`} onChange={this.handleChange} />
+                                    </Col>
+                            </FormGroup>}
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="select">Observación</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="note-input" name="note" value={this.state.authorization.note} onChange={this.handleChange} />
```

Comment:
`Warning: Failed prop type: You provided a 'value' prop to a form field without an 'onChange' handler. This will render a read-only field. If the field should be mutable use 'defaultValue'. Otherwise, set either 'onChange' or 'readOnly'.`

Idem anterior.

Reply:
Porque me olvidé el handle change jeje